ui_framework/src/components/table/listing_table/listing_table_toolbar.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiToolBarSection,
+} from '../../';
+
+export function KuiListingTableToolbar({ pagerComponent, actionComponent, onFilter }) {
+  return (
+    <KuiToolBar>
+      <KuiToolBarSearchBox onFilter={onFilter} />
+
+      <KuiToolBarSection>
+        {actionComponent}
+      </KuiToolBarSection>
+
+      <KuiToolBarSection>
+        {pagerComponent}
+      </KuiToolBarSection>
+    </KuiToolBar>
+  );
+}
+
+KuiListingTableToolbar.propTypes = {
+  onFilter: PropTypes.func.isRequired,
+  pagerComponent: PropTypes.node,
```

Comment:
I think we can drop the "Component" from names and just make this `pager` and `actionComponent` can become `actions`. Otherwise we're sort of implementing Hungarian notation.